src/linuxthreads.c

```diff
@@ -609,16 +618,22 @@ int ListAllProcessThreads(void *parameter,
 #ifndef PR_SET_PTRACER
 # define PR_SET_PTRACER 0x59616d61
 #endif
-    sys_prctl(PR_SET_PTRACER, clone_pid);
+    sys_prctl(PR_SET_PTRACER, clone_pid, 0, 0, 0);
 
     sys_sigprocmask(SIG_SETMASK, &sig_old, &sig_old);
 
     if (clone_pid >= 0) {
       int status, rc;
+      #if defined (__aarch64__)
+      while ((rc = sys_waitpid(clone_pid, &status, __WALL)) < 0 && ERRNO == EINTR) {
+             /* Keep waiting                                                 */
+      }
+      #else
```

Comment:
Why does this need to be different?

Reply:
One is `sys_waitpid()` and another is `sys0_waitpid()`. I don't recall why I added It.
Original code works on the both platforms, so I will revert the change.